docs/source/en/index.mdx

```diff
@@ -48,6 +48,7 @@ available a colab notebook to directly try them out.
 | [score_sde_ve](./api/pipelines/score_sde_ve) | [**Score-Based Generative Modeling through Stochastic Differential Equations**](https://openreview.net/forum?id=PxTIG12RRHS) | Unconditional Image Generation | 
 | [score_sde_vp](./api/pipelines/score_sde_vp) | [**Score-Based Generative Modeling through Stochastic Differential Equations**](https://openreview.net/forum?id=PxTIG12RRHS) | Unconditional Image Generation | 
 | [stable_diffusion](./api/pipelines/stable_diffusion/text2img) | [**Stable Diffusion**](https://stability.ai/blog/stable-diffusion-public-release) | Text-to-Image Generation | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/training_example.ipynb)
+| [stable_diffusion](./api/pipelines/stable_diffusion/panorama) | [**MultiDiffusion**](https://multidiffusion.github.io/) | Text-to-Panorama Generation |
```

Comment:
Feel free to also add the pipeline to the overview.mdx of Stable Diffusion as shown here: https://github.com/huggingface/diffusers/pull/2040/files#diff-fda000f4c90827a6a9b7776eec6f151191c1eda52c2f2e590be6223d88a07ebe

<img width="1847" alt="image" src="https://user-images.githubusercontent.com/22957388/219695094-d19380f2-992e-4b2c-9cf9-6c271b9b2419.png">